lib/msf/core/payload/adapter/fetch/windows_options.rb

```diff
@@ -0,0 +1,12 @@
+module Msf::Payload::Adapter::Fetch::WindowsOptions
+
+  def initialize(*args)
+    super
+    register_options(
+      [
+        Msf::OptEnum.new('FETCH_COMMAND', [true, 'Command to fetch payload', 'CURL', %w{ CURL TFTP CERTUTIL }]),
```

Comment:
Since CURL isn't available by default and CERTUTIL is, would this make sense to default to CERTUTIL?

Reply:
All versions of Windows since 8.1 have cURL natively, so all Windows released in the last 8 years have it.  I chose cURL as the default across both platforms because cURL is mostly universal across platform, but more importantly, cURL works with every server.